### DISCUSSION TITLE
first step failed - plz help

### DISCUSSION THREAD

**Discussion by thelaneburch** [ordinals/ord#1631] [General]
I got to the bitcoin.conf within the bitcoin core application and saved txindex=1 in the file. I had pruning turned on... how am I able to reset the application? does this lose the BTC in my wallet? Is bitcoin core able to verify transactions off of an external hard drive with the Bitcoin Blockchain?

**Comment by goddesslens** [comment]
1)In any case back up your seedphrase. 
2)Opening bitcoind in the cmd/terminal with -txindex is a better choice.
3)Pruned bitcoin node cannot inscribe ordinals with ord wallet.
4) As long as you mention the directory , bitcoind can index transaction

Consult an expert

**Reply by thelaneburch** [reply]
That's why I'm here!
